### DISCUSSION TITLE
Postgresql Major updates

### DISCUSSION THREAD

**Discussion by Tilusch** [cloudnative-pg/cloudnative-pg#5753] [Q&A] [answered]
In the current state the CloudNative-PG oeprator does not support Major updates without re creating a db. I know its a good practice to test a new Postgresql version on a new sepperate Cluster wich one could do on a sepperate testing environment. Non the less would it be a great feature to be able to rollout major updates with the operator, is this feature on the roadmap or will this never be implemented?

**Comment by saolof** [comment, marked as answer]
Seems to be in the 1.26 milestone on their roadmap: https://github.com/orgs/cloudnative-pg/projects/1/views/1
